statistics.md

```diff
@@ -0,0 +1,96 @@
+# Collecting user interactions
+[Back to the list of all defined endpoints](endpoints.md)
+
+## View events endpoint
+**POST /api/statistics/viewevents**
+
+To track a page view, make an HTTP POST request to this endpoint.
+
+The sections below describe the parameters for a page view event. All other information about the interaction can be derived from the request itself.
+
+### Required Parameters
+- `targetId`: The id of the object the user viewed
+- `targetType`: The type of the object the user viewed
+
+### Status codes:
+
+- `201` Created: if the operation succeeded
+- `400` Bad request: if any of the parameters are missing or invalid.
+- `422` Unprocessable Entity: if the parameters don't refer to a valid object.
+
+### Example item page view:
+
+```json
+{
+  "targetId": "43f9bb3e-f90d-458f-9858-7e4589481d18",
+  "targetType": "item"
+}
+```
+
+
+## Search events Endpoint
+**POST /api/statistics/searchevents**
+
+To track a search, make an HTTP POST request to this endpoint.
+
+The sections below describe the parameters for a search event. All other information about the interaction can be derived from the request itself.
+
+### Required Parameters
+- `query`: The discovery search string.
+- `page`: An object that describes the pagination status. For more information take a look at the [pagination documentation](README.md#Pagination).
+- `sort`: An object that describes the sort status. For more information take a look at the [pagination documentation](README.md#Pagination).
+
+### Optional Parameters
+- `dsoType`: Limits the search to a specific DSpace Object type. Possible values:
+     - `all`
+     - `item`
+     - `community`
+     - `collection`
+- `scope`: The UUID of a specific DSpace container (site, community or collection) to which the search has to be limited.
+- `configuration`: The name of a Discovery configuration that was used by the search.
+- `appliedFilters`: An array of search filters  used to filter the result set. For more information take a look at the [search documentation](search-endpoint.md#matching-dspace-objects-search-results).
+
+### Status codes:
+
+- `201` Created: if the operation succeeded
+- `400` Bad request: if any of the required parameters are missing, or any parameters are invalid.
```

Comment:
can you add 422 also here for the case where the parameter dsoType is syntactically valid but lead to a not existing object?